### ISSUE TITLE
Terraform init stoped working!

### ISSUE THREAD

**Issue by AmitThakkar** [terraform-aws-modules/terraform-aws-eks#676]
# I have issues
`terrform init` was working yesterday, giving below error today:
```
amikum$ terraform init
Initializing modules...
Downloading terraform-aws-modules/eks/aws 8.0.0 for eks...

Error: Failed to download module

Could not download module "eks" (eks.tf:55) source code from
"https://api.github.com/repos/terraform-aws-modules/terraform-aws-eks/tarball/v8.0.0//*?archive=tar.gz":
Error opening a gzip reader for
/var/folders/zy/pqcpy3lj56x_v0tj55rfwwq00000gq/T/getter584767963/archive: EOF.


Error: Failed to download module

Could not download module "eks" (eks.tf:55) source code from
"https://api.github.com/repos/terraform-aws-modules/terraform-aws-eks/tarball/v8.0.0//*?archive=tar.gz":
Error opening a gzip reader for
/var/folders/zy/pqcpy3lj56x_v0tj55rfwwq00000gq/T/getter584767963/archive: EOF.
```

## I'm submitting a...

* bug report
* [ ] feature request
* [ ] support request - read the [FAQ](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/faq.md) first!
* [ ] kudos, thank you, warm fuzzy

## What is the current behavior?
Getting below Error:
```
Error: Failed to download module

Could not download module "eks" (eks.tf:55) source code from
"https://api.github.com/repos/terraform-aws-modules/terraform-aws-eks/tarball/v8.0.0//*?archive=tar.gz":
Error opening a gzip reader for
/var/folders/zy/pqcpy3lj56x_v0tj55rfwwq00000gq/T/getter584767963/archive: EOF.
```

## What's the expected behavior?
It should work.

## Environment details

* Affected module version:
* OS:
* Terraform version:

**Comment by dpiddockcmp** [comment]
Lots of reports of this in various modules. Hashicorp have screwed something up and the only known fix is to use Terraform 0.12.19

Duplicate of #672 and #674

**Comment by AmitThakkar** [comment]
@dpiddockcmp Thank you very much for your quick help. With Terraform 0.12.19 working again.
